src/styles/main.scss

```diff
@@ -507,12 +507,37 @@ h6{
 }
 
 
-/*
-*  Experiment Key: dd_add_copy_code_button_to_docs
-*  https://app.optimizely.com/v2/projects/16812830475/experiments/20580130094/variations
-*/
+.code-badge {
+	opacity: 0!important;
```

Comment:
use of `!important` is how highlight js extension recommends to add styling tweaks much to the chagrin CSS enthusiasts everywhere.